circle.yml

```diff
@@ -32,6 +32,7 @@ jobs:
               cp .env.default .env
               composer install -d programs-remote-tests/
               chown noroot:www-data /var/www/html
+              su noroot -c '/usr/local/bin/wait-db.sh'
```

Comment:
Where is this `wait-db.sh`? Is it a CircleCI thing?
I couldn't find it anywhere, but I do see it's being used in `dev up --build`

Reply:
I found the script, is it a docker thing?

`/usr/local/bin/wait-db.sh`
```
#!/bin/sh
until mysql.sh -e "show databases;" 2> /dev/null > /dev/null
do
    echo -n .
    sleep 1
done
echo
```

Reply:
ok, I found it. I'm happy now
https://github.com/retreatguru/docker/blob/master/programs/bin/wait-db.sh